File: cs/Iris/ExpressionElement/Instruction/PlayMediaInstruction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Iris.SimpleExpressionElement;
using Iris.ParserUtils;

namespace Iris.ExpressionElement.Instruction
{
    public class PlayMediaInstruction : InstructionExpression
    {
        //public PathExpression path;
        public string name;
        public LiteralExpression loop = new LiteralExpression(new Token(Token.TokenValue.BOOLEAN, false));

        public PlayMediaInstruction()
        {

        }
    }
}

File: java/Iris/src/com/rokuan/iris/instruction/PlayMediaInstruction.java
package com.rokuan.iris.instruction;

import com.rokuan.iris.expression.LiteralExpression;
import com.rokuan.iris.parser.Token;

public class PlayMediaInstruction extends InstructionExpression
{
	//public PathExpression path;
	public String name;
	public LiteralExpression loop = new LiteralExpression(new Token(Token.TokenValue.BOOLEAN, false));

	public PlayMediaInstruction()
	{

	}
}
